packages/scandipwa/src/route/ProductPage/ProductPage.container.js

```diff
@@ -292,13 +292,13 @@ export class ProductPageContainer extends PureComponent {
     isProductInformationTabEmpty() {
         const dataSource = this.getDataSource();
 
-        return !dataSource?.description?.html.length;
+        return !!dataSource?.description?.html.length;
     }
 
     isProductAttributesTabEmpty() {
         const dataSource = this.getDataSource();
 
-        return Object.keys(dataSource?.attributes || {}).length === 0;
+        return !!Object.keys(dataSource?.attributes || {}).length;
```

Comment:
I think converting a number to boolean here has some disadvantages.
1. It's harder to read. Especially for newbies.
2. It's not huge, but can impact performance since the JS engine will have to convert types instead of simply comparing them.